### PR TITLE
compat: use tarantool.io URLs for compat options

### PR DESCRIPTION
**Pull request by Totktonada**
We plan to provide compat options information via the website. It was requested by Kirill Yukhin instead of the wiki pages (and I don't mind it).

### CODE REVIEW

File: src/lua/compat.lua [hunk 1]
```diff
@@ -90,7 +90,7 @@ local options_order = { }
 
 local help = [[
 Tarantool compatibility module.
-To get help, see the Tarantool manual at https://github.com/tarantool/tarantool/wiki/compat .
+To get help, see the Tarantool manual at https://tarantool.io/compat
```
Comment (Totktonada) [inline src/lua/compat.lua:93]: @NickVolynkin Highlighting your attention on this page just in case.

Comment (Totktonada) [inline src/lua/compat.lua:93]: NB: Adjust this PR if #8283 lands first.

Comment (Totktonada) [inline src/lua/compat.lua:93]: NB: Adjust this PR if #8338 lands first.

**Review by lastoCHka42 — APPROVED** [review]
Website with redirects deployed

File: src/lua/compat.lua (hunk 1 above)
Comment (Totktonada) [reply src/lua/compat.lua:93]: It works, closing.

Comment (Totktonada) [reply src/lua/compat.lua:93]: Commented in PR #8283.

Comment (Totktonada) [reply src/lua/compat.lua:93]: Commented in PR #8338.

**Pull request merged**